src/main/webapp/src/app/common/growpod-page-styles.css

```diff
@@ -9,3 +9,21 @@
 .text-center {
   text-align: center;
 }
+
+.profile-pic-container-s {
+  position: relative;
+  width: 33%; /* The size you want */
+  margin-left: auto;
+  margin-right: auto;
+  background-color: #fff;
+  border: 4px solid black;
+  border-radius: 50%;
+  -moz-border-radius: 50%;
+  -webkit-border-radius: 50%;
+}
+
+.profile-pic-container-s:after {
+  content: '';
```

Comment:
What is the intention with this line?

Reply:
Without this line, the circle's height is not the same as its width.